src/common/templates/check-your-answers.njk

```diff
@@ -2,149 +2,66 @@
     <dl class="app-check-your-answers app-check-your-answers--short" style="margin-bottom: 0;">
       <div class="app-check-your-answers__contents">
         <dt style="width: 200px; border-bottom: none;" class="govuk-heading-m app-check-your-answers__question">
-          {{ __('label_departure_details') }}
+          {{ __('heading_departure_details') }}
         </dt>
         {% if showChangeLinks %}
         <dd style="border-bottom: none;" class="app-check-your-answers__change">
           <a href="/garfile/departure" id='departure-change'>
-            Change<span class="govuk-visually-hidden">Departure</span>
+            {{__('change_prefix')}}<span class="govuk-visually-hidden">{{ __('heading_departure_details') }}</span>
           </a>
         </dd>
         {% endif %}
       </div>
     </dl>
 
     <dl class="app-check-your-answers app-check-your-answers--short">
-      <div class="app-check-your-answers__contents">
-        <dt class="app-check-your-answers__question">{{ __('field_departure_date') }}</dt>
-        <dd class="app-check-your-answers__answer">{{garfile.departureDate}}</dd>
-        <dd class="app-check-your-answers__change"></dd>
-      </div>
-      <div class="app-check-your-answers__contents">
-        <dt class="app-check-your-answers__question">{{ __('field_departure_time') }}</dt>
-        <dd class="app-check-your-answers__answer">{{garfile.departureTime}}</dd>
-        <dd class="app-check-your-answers__change"></dd>
-      </div>
-      <div class="app-check-your-answers__contents">
-        <dt class="app-check-your-answers__question">{{__('field_departure_port')}}</dt>
-        <dd class="app-check-your-answers__answer">{{garfile.departurePort | upper}}</dd>
-        <dd class="app-check-your-answers__change"></dd>
-      </div>
-      <div class="app-check-your-answers__contents">
-        <dt class="app-check-your-answers__question">Departure port coordinates</dt>
-        <dd class="app-check-your-answers__answer">
-          {{__('field_latitude')}}: {{garfile.departureLat}}<br>
-          {{__('field_longitude')}}: {{garfile.departureLong}}</dd>
-        <dd class="app-check-your-answers__change">
-        </dd>
-      </div>
```

Comment:
Using a macro because the rows are all virtually the same, this reduces a lot of the markup on this page.